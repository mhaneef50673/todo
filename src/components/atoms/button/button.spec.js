import React from 'react';
import { mount } from 'enzyme';
import { mountToJson} from 'enzyme-to-json';
import Button from './index';

describe('Button Component', () => {
  it('should render the component', () => {
    const tree = mount(
      <Button buttonText="Add Item" extraClass="extraClass" />
    )
    expect(mountToJson(tree)).toMatchSnapshot();
  });
});