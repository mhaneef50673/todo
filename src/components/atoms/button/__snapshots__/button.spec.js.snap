// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component should render the component 1`] = `
<Button
  action={[Function]}
  buttonText="Add Item"
  extraClass="extraClass"
>
  <button
    className="extraClass"
    type="button"
  >
    Add Item
  </button>
</Button>
`;
